.github/workflows/build-docker-from-tag.yml

```diff
@@ -1,5 +1,17 @@
 name: Build Image from tag
 on:
+  workflow_call:
```

Comment:
Should we allow users to override the tag?

Reply:
If you call that manually the tag will be empty, so user have to supply the tag.